src/main/java/net/dv8tion/jda/core/requests/restaction/MessageAction.java

```diff
@@ -505,6 +505,22 @@ public MessageAction override(final boolean bool)
         return this;
     }
 
+    /**
+     * Sets the {@link net.dv8tion.jda.core.entities.Message.Activity Activity} and its {@link net.dv8tion.jda.core.entities.Message.Application Application}
+     * that should be used for this Message.
+     * <br><strong>Be careful with this feature!</strong>
```

Comment:
What's the point of this warning?

Reply:
JDA does not provide session ids from Rich Presence yet that users can use and received activity is not send-able yet